fuse/data/datasets/dataset_default.py

```diff
@@ -199,10 +200,11 @@ def _getitem_multiprocess(item: Union[Hashable, int, np.integer]):
         return dataset.getitem(item, **kwargs)
 
 
-    def get_multi(self, items: Optional[Sequence[Union[int, Hashable]]] = None, workers: int = 10, verbose: int = 1, **kwargs) -> List[Dict]:
+    def get_multi(self, items: Optional[Sequence[Union[int, Hashable]]] = None, workers: int = 10, verbose: int = 1, mp_context: Optional[str] = None, **kwargs) -> List[Dict]:
         """
         See super class
-        :param workers: number of processes to read the data. set to 0 for a single process.  
+        :param workers: number of processes to read the data. set to 0 for a single process.
```

Comment:
-> "set to 0 to not use multi processing (useful when debugging)"

maybe a subtlety but when set to 1 you actually do create a single process that reads the data, which isn't the main process.